packages/web/components/DataTable/index.tsx

```diff
@@ -246,17 +251,20 @@ export default function DataTable({
       label,
       options: {
         filter,
+        display,
         filterList: cFilterList,
         customFilterListOptions: {
           render: filterSelectionRender,
         },
         filterOptions: {
           names: filterNames,
+          ...cFilterOptions,
         },
         sort,
         sortDirection: getSortDirection(order, name),
         searchable,
         customBodyRender: actions ? actionsColumnRender(actions) : customColumnRender,
+        ...cOptions,
```

Comment:
place this spread at the top of the object, we consider all the specializations as a higher priority, placing at the top they won't be overridden...